lib/main.ts

```diff
@@ -174,6 +173,39 @@ export class Transpiler {
     return this.output.getResult();
   }
 
+  private recursiveMkdirSync(p: string) {
```

Comment:
should be static, since it does not reference any class members in the implementation